mcp_server/pkg/internalapi/config.go

```diff
@@ -54,6 +54,10 @@ var (
 		"INTERNAL_API_URL_RBAC",
 		"MCP_RBAC_GRPC_ENDPOINT",
 	}
+	FeatureHubEndpoint = []string{
```

Comment:
@DamjanBecirovic this is not used in `LoadConfig()` below and we should probably follow the above pattern to name this const  like `rbacEndpointEnvs` etc